packages/react-components/src/templates/__tests__/UserProfileHeader.test.tsx

```diff
@@ -138,3 +142,39 @@ it('displays number of shared research', async () => {
   );
   expect(screen.getByText('Shared Outputs (5)')).toBeInTheDocument();
 });
+
+it('renders the navigation for active and inactive groups', () => {
+  render(<UserProfileHeader {...boilerplateProps} />);
+
+  expect(
```

Comment:
Please add a regression test test to ensure these tabs are not displayed when the feature is disabled. Include ((REGRESSION)) in the title like other such tests to help when we remove the feature flag